.eslintrc

```diff
@@ -7,14 +7,15 @@
     "browser": true
   },
   "rules": {
-    "react/jsx-props-no-spreading": { "custom": "ignore" },
-    "react/prop-types": "off",
-    "no-nested-ternary": "off",
+    "react/jsx-props-no-spreading": 0, // disables prop spreading messages in src/components/link.jsx
```

Comment:
Why were  `{ "custom": "ignore" }` and `"off"` changed to 0?

Reply:
Also `{ "custom": "ignore" }` only applies to custom components, so it's more specific than just allowing prop spreading in general. For instance `<img {...props} />` would still not be allowed under this rule

Reply:
I was getting errors when formatting the code using the ESLint extension with this configuration until I changed both to 0.

Reply:
What were the errors?

Reply:
Weird, that syntax is in the docs. I guess we should use 0 consistently instead of "off" then? We can change `"no-nested-ternary": "off"`

Reply:
Just saw your other comment - does `off` work? Just not `custom: ignore`?

Reply:
`"off"` and `0` work the same. However,  `{ "custom": "ignore" }` does not work. I will change all the `"off"` to be `0`.

Reply:
If `off` works then maybe we should use that since it's more clear than 0. We should just be consistent about which one we use

Reply:
Changed all `0`s to `"off"`.